### PR TITLE
Create a pr.yml

### PR DESCRIPTION
**Pull request by AdrianValdes**
hi @paschalidi,
I set it up and run the tests. I checked the actions on GitHub and it is all working as far as my knowledge tells me.

### CODE REVIEW

**Comment by AdrianValdes** [comment]
hi @paschalidi, 
I set it up and run the tests. I checked the actions on GitHub and it is all working as far as my knowledge tells me.

**Comment by paschalidi** [comment]
>hi @paschalidi,
I set it up and run the tests. I checked the actions on GitHub and it is all working as far as my knowledge tells me. (My knowledge on this topic is very low, I know :)) 

I would say this should be your description and not a message that comes after the description. Now the description is empty.

**Comment by AdrianValdes** [comment]
I moved it up to the description.

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
```
Comment (paschalidi) [inline .github/workflows/pr.yml:8]:
```suggestion
  lint:
```

We dont build anything just yet. We only lint

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      with:
+        node-version: '15.x'
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Bootstrap
+      run: npm run bootstrap  
+
+    - name: Lint types and code
```
Comment (paschalidi) [inline .github/workflows/pr.yml:25]: "Lint types" sounds odd. Can you rephrase a little bit please?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # core
-an open source project that builds free small advertisments
+an open source project that builds free small advertisements
```
Comment (paschalidi) [inline README.md:2]: 👍

File: .github/workflows/pr.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '15.x'
+    
+    - name: Install dependencies
+      run: npm ci
```
Comment (paschalidi) [inline .github/workflows/pr.yml:20]: Good thinking in using `ci` here!

**Review by paschalidi — COMMENTED** [review]
Looks good @AdrianValdes. I got some comments

File: .github/workflows/pr.yml (hunk 2 above)
Comment (AdrianValdes) [reply .github/workflows/pr.yml:25]: Is `Lints and formats code` better?

I copied the `Lint types and code` from Chakra, I thought it was a community definiton.

File: .github/workflows/pr.yml (hunk 4 above)
Comment (AdrianValdes) [reply .github/workflows/pr.yml:20]: I was reading about it and apparently is faster and better than `i` :)

File: .github/workflows/pr.yml (hunk 2 above)
Comment (AdrianValdes) [reply .github/workflows/pr.yml:25]: Or maybe just `Lint`

Comment (paschalidi) [reply .github/workflows/pr.yml]: > Is Lints and formats code better? 

We arent doing _any_ formating here. We only checking if a developer is pushing code that it isnt following our eslint rules

Comment (paschalidi) [reply .github/workflows/pr.yml]: Just Lint sounds good to me

**Comment by paschalidi** [comment]
There is warning on the stories about that alert. We can disable eslint for that specific case since basically we are only doing it for the stories to have a more meaningful presentation

**Comment by AdrianValdes** [comment]
Oh, it bothers you, hehe, me too :) I am on it!

**Review by paschalidi — APPROVED** [review]
Merge this in!

**Comment by paschalidi** [comment]
Need to get you a new task. I am gonna write one very quickly.
